app/views/classified_listings/_form.html.erb

```diff
@@ -32,10 +32,6 @@
             <summary>Category details/rules</summary>
           </details>
         </div>
-        <div class="field">
-          <%= form.label "tags_list", "Tags" %>
-          <%= form.text_field "tags_list", placeholder: "8 tags max, comma separated, no spaces or special characters" %>
-        </div>
```

Comment:
Tagged you specifically @benhalpern to make sure this wasn't intentional for some caching sorcery or service worker wizardry that I'm unaware of. This form field gets overridden by the `Tags` component loaded in from `listingForm`.

Reply:
This is actually used, essentially as a placeholder while the rest loads so that content doesn't jump. This is how we do SSR... Not so sophisticated at the moment.

I'd say that this was a hack we could remove and instead just display a loader for the whole form... But we shouldn't just remove this on it's own.

Otherwise PR does look great. Good instincts asking.

Reply:
That makes sense! I actually had that exact thought about it being a placeholder to prevent screen jumping. I refreshed locally a bunch of times (obviously that's not the best proxy for prod behavior) and I saw that it was super snappy (no jumping) still so I figured I'd ask.

I put the form field back in and updated it to be "correct".

Reply:
Could you add a comment to it? Since it looks like a specific situation that some wouldn't figure out (aka me, I totally would have missed that haha), it'd be awesome to have a couple of lines before that block explaining why it has to stay there:

```erb
<%# even though the Tags component overrides this block, this is here because etc etc %>
```

something like that :D 

What do you think?

Reply:
Makes sense! Updated.